nio.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "vullk4n",
+			"oem": "Motorola",
+			"device": "Moto G100",
+			"filename": "crDroidAndroid-15.0-20241221-nio-v11.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/nio/11.x/crDroidAndroid-15.0-20241221-nio-v11.0.zip/download",
+			"timestamp": 1734750414,
+			"md5": "20877f70b0fc4de0c72a5c5aaef7b791",
+			"sha256": "a89092411084d2802b41d4b1f28f6aa47ead1ab8d46376830d9b068ddd108160",
+			"size": 1261955091,
+			"version": "11.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-official-15-0-moto-g100-crdroid-11-0-monthly-2024-12-21.4709072/",
```

Comment:
fix url for crDroid Device Group and Support Community
also fix install page url - points to 10 not 11
and dl page should be crdroid.net/nio/11